haystack/schema.py

```diff
@@ -94,17 +96,52 @@ def __str__(self):
         return str(self.to_dict())
 
 
+@dataclass
+class Answer:
+    answer: Optional[str]
+    type: Literal["generative", "extractive"]
+    #question: Optional[str]
+    score: Optional[float]
```

Comment:
score should be mandatory

Reply:
@oryx1729 @julian-risch What was again your argument for making it mandatory in our discussion? Makes sense for any retrieved documents but what about a fresh Document that just gets created in a FileConverter? Either we keep it optional there or we add a default `0.0`

Reply:
I think `Optional` is better. The value `0.0` could be misleading.